### PR TITLE
fix(multi): missing param for Multi's constructor

### PR DESCRIPTION
**Pull request by rgs1**
Signed-off-by: Raul Gutierrez S <rgs@twitter.com>

### CODE REVIEW

File: zktraffic/tests/test_packets.py [hunk 1]
```diff
@@ -86,3 +86,11 @@ def test_connects():
 
   assert stats.global_stats.by_op_counters[OpCodes.CONNECT] == 3
   assert stats.global_stats.by_op_counters[OpCodes.CLOSE] == 3
+
+
+def test_multi():
+  zkt, stats = default_zkt()
+  consume_packets('multi', zkt)
+
+  print(stats.global_stats.by_op_counters)
```
Comment (Yasumoto) [inline zktraffic/tests/test_packets.py]: delete debugging print

Comment (chchen) [inline zktraffic/tests/test_packets.py]: 2nded

Comment (rgs1) [inline zktraffic/tests/test_packets.py]: oops!

**Comment by rgs1** [comment]
il est maintenant fixé, beau monsieur!

**Pull request merged**